app/src/blog/views.py

```diff
@@ -1,8 +1,9 @@
-from django.views.decorators.http import require_http_methods
 from django.contrib.admin.views.decorators import staff_member_required
-from django.shortcuts import render, redirect
-from .models import BlogPost
+from django.shortcuts import redirect, render
+from django.views.decorators.http import require_http_methods
+
 from .forms import BlogPostForm
+from .models import BlogPost
```

Comment:
I'm not an Python expert or even a Python developer, but shouldn't we prefer absolute imports over relative ones? Again I'm not the expert and I don't know which one is "cleaner".

Reply:
PEP8 states that ["Absolute imports are recommended, (...) however, explicit relative imports are an acceptable alternative to absolute imports (...)"](https://www.python.org/dev/peps/pep-0008/#imports). I tend to convert relative imports to absolute ones, but PyCharm possibly tries to match new imports to existing ones.

Reply:
I think I can change (almost) all relative imports to absolute, but this change could be applied directly on master.